BenchmarkComponent/BenchmarkComponent.cpp

```diff
@@ -61,4 +65,63 @@ namespace winrt::BenchmarkComponent::implementation
     void ClassWithMultipleInterfaces::DefaultDoubleProperty(double val)
     {
     }
+
+    ClassWithMarshalingRoutines::ClassWithMarshalingRoutines()
+    {
+    }
+
+    hstring ClassWithMarshalingRoutines::DefaultStringProperty()
+    {
+        return hstring();
+    }
+
+    void ClassWithMarshalingRoutines::DefaultStringProperty(hstring val)
+    {
+    }
+    
+    Windows::Foundation::IInspectable ClassWithMarshalingRoutines::DefaultKeyValuePairProperty()

```

Comment:
consider DefaultKeyValuePairPropertyAsInspectable or DefaultKeyValuePairPropertyTypeErased or something as I expect we might want equivalents which are not type erased.

Reply:
Makes sense.